lib/run.js

```diff
@@ -155,9 +160,10 @@ function run (opts, cb) {
       let result = {
         title: opts.title,
         url: opts.url,
-        requests: histAsObj(requests, totalRequests),
+        requests: histAsObj(requests, totalCompletedRequests),
         latency: addPercentiles(latencies, histAsObj(latencies)),
         throughput: histAsObj(throughput, totalBytes),
+        sent: totalRequests,
```

Comment:
Ohh. I forgot to put the helper lib hdr-histogram-percentiles-obj in, is it okay if I do that too?